content/templates.xql

```diff
@@ -65,7 +66,7 @@ declare function templates:apply($content as node()+, $resolver as function(xs:s
  : @param $configuration a map of configuration parameters. For example you may provide a
  :  'parameter value resolver' by mapping $templates:CONFIG_PARAM_RESOLVER to a function
  :  whoose job it is to provide values for templated parameters. The function signature for
- :  the 'parameter value resolver' is f($param-name as xs:string) as item()*
+ :  the 'parameter value resolver' is either f($param-name as xs:string) as item()*.
```

Comment:
the original text disappeared: https://github.com/eXist-db/shared-resources/pull/14/commits/613f3fd4c192ba8af0817f57fd44d5542aa65577 

"the 'parameter value resolver' is either f($param-name as xs:string) as item()*, or f($param-name as xs:string, $model as map(*)?) as item()*."